IntegrationTests/Services/AWSS3IntegrationTests/S3ConcurrentTests.swift

```diff
@@ -18,38 +18,27 @@ final class S3ConcurrentTests: S3XCTestCase {
     // Payload just below chunked threshold
     // Tests concurrent upload of simple data payloads
     func test_10x_1MB_getObject() async throws {
-        fileData = try generateDummyTextData(count: CHUNKED_THRESHOLD - 1)
+        fileData = S3ConcurrentTests.generateRandomTextData(ofSizeInMB: 1)
         try await repeatConcurrentlyWithArgs(count: 10, test: getObject, args: fileData!)
     }
 
     // Payload at chunked threshold, just large enough to chunk
     // Tests concurrent upload with aws-chunked encoding & flexible checksums
     func test_10x_1_5MB_getObject() async throws {
-        fileData = try generateDummyTextData(count: CHUNKED_THRESHOLD)
+        fileData = S3ConcurrentTests.generateRandomTextData(ofSizeInMB: 1.5)
         try await repeatConcurrentlyWithArgs(count: 10, test: getObject, args: fileData!)
     }
 
     // Payload 256 bytes with 200 concurrent requests, sends as simple data
     // Tests very high concurrency with small data payloads
     func test_200x_256B_getObject() async throws {
-        fileData = try generateDummyTextData(count: 256)
+        let bytes_256 = Double(256) / (1024 * 1024)
```

Comment:
Any Swift literal with a decimal point is a Double.  So you can just use `256.0` or `256.` instead of the Double initializer.